src/components/scheduling/SchedulingGoalsPanel.svelte

```diff
@@ -40,7 +40,7 @@
   let visibleSchedulingGoalSpecs: SchedulingGoalPlanSpecification[] = [];
 
   // TODO: remove this after db merge as it becomes redundant
-  $: visibleSchedulingGoalSpecs = $allowedSchedulingGoalSpecs.filter(({ goal_metadata: goalMetadata }) => {
+  $: visibleSchedulingGoalSpecs = $schedulingGoalInvocations.filter(({ goal_metadata: goalMetadata }) => {
```

Comment:
Does `schedulingGoalInvocations` only contain goals that the user has access to similar to `allowedSchedulingGoalSpecs`?

Reply:
Change has been made, `allowedSchedulingGoalInvocations` now exists and follows the same filtering pattern.